samples/docker/alpine/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/dotnet/aspnet:7.0-alpine AS base
+WORKDIR /app
+EXPOSE 80
+ENV ASPNETCORE_URLS=http://+:80
+
+FROM mcr.microsoft.com/dotnet/sdk:7.0-alpine AS build 
+
+WORKDIR /src
+COPY . ./
+RUN dotnet restore
+RUN dotnet publish -c Release -o /app/publish
+
+FROM base AS final
+
+# download and install the New Relic .NET Agent using the tarball
+# find the latest version at https://download.newrelic.com/dot_net_agent/latest_release and update the filename as needed
+# the agent is installed at /usr/local/new-relic-dotnet-agent
+RUN apk update && apk add --no-cache wget tar \
+    && wget https://download.newrelic.com/dot_net_agent/latest_release/newrelic-dotnet-agent_10.15.0_amd64.tar.gz \
```

Comment:
I'm trying to think of a way we could keep this somewhat up to date... Part of the release process? Tech debt ticket?

Reply:
Personally, I'd like to see some kind of symlink (or equivalent) on the download site that is an unchanging URL that always points to the latest version... ;-)

Reply:
As noted in team pairing, since the URL is for the `latest_release` folder but has a specific version named, this will break the next time we do a release.  I'll work on a more automagical way of doing things.

Reply:
Resolved with a little `wget` magic.